common/src/main/scala/explore/components/Tile.scala

```diff
@@ -3,17 +3,30 @@
 
 package explore.components
 
+import cats.syntax.all._
 import explore.Icons
 import explore.components.ui.ExploreStyles
+import explore.model.enum.TileSizeState
 import japgolly.scalajs.react._
 import japgolly.scalajs.react.vdom.html_<^._
 import react.common._
+import react.common.implicits._
 import react.semanticui.collections.menu._
+import react.semanticui.elements.button.Button
 
 final case class TileButton(body: VdomNode)
 
-final case class Tile(title: String, movable: Boolean, back: Option[TileButton])
-    extends ReactPropsWithChildren[Tile](Tile.component)
+final case class Tile(
```

Comment:
I understand that all our tiles are unmovable now, is that right?

Reply:
I'd expect yes but we don't need to specify that in the tile but in the layout. The flag here was to put an icon to use as a handle
but that is not needed anymore